client/layout/guided-tours/tours/main-tour.js

```diff
@@ -96,9 +97,9 @@ export const MainTour = makeTour(
 		>
 			<p>{ translate( 'Want to take a look at your site?' ) }</p>
 			<Continue click step="view-site" target="sitePreview">
-				{ translate( 'Click {{strong}}View Site{{/strong}} to continue.', {
+				{ translate( 'Click {{viewSiteButton/}} to continue.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Click {{ViewSiteButton/}} to continue.' )` *ES Score: 25*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).